pyproject.toml

```diff
@@ -41,6 +39,7 @@ dev = [
 test = [
     "pytest>=6.0",
     "pytest-cov",
+    "nbproject",
```

Comment:
To be honest, I'm not sure why you need `nbproject` at all. For tests, you only need `nbproject_test`.

It's used once in the code:

`from nbproject._logger import logger` (https://github.com/scverse/pytometry/blob/main/tests/test_notebooks.py#L4) and this can easily be replaced with just a print or Python logger statement